packages/gasket-plugin-intl/lib/utils.js

```diff
@@ -8,11 +8,13 @@ function getIntlConfig(gasket) {
 }
 
 function getAssetPrefix(gasket) {
-  const { next = {}, intl = {} } = gasket.config;
-  let { assetPrefix = '' } = next;
+  const { next = {}, intl = {}, zone } = gasket.config;
+  let assetPrefix = zone || next.assetPrefix || '';
```

Comment:
Just realized this should be switched
```suggestion
  let assetPrefix = next.assetPrefix || zone || '';
```
since `next.assetPrefix` should override `zone` if set.